src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -121,8 +120,9 @@
                          chat-id show-input? message-pin-enabled edit-enabled in-pinned-view? can-delete-message-for-everyone?]}]
   [rn/view {:style (when (and platform/android? (not in-pinned-view?)) {:scaleY -1})}
    (if (= type :datemark)
-     ;; TODO (flexsurfer) implement and use date divider component https://github.com/status-im/status-mobile/issues/14523
-     [message-datemark/chat-datemark (:value message)]
+     [rn/touchable-without-feedback
+      {:on-press #(rn/dismiss-keyboard!)}
```

Comment:
[Nitpick] You can avoid the extra anonymous function as you did in `status-im2.contexts.quo-preview.dividers.date`.